tests/test_project.py

```diff
@@ -61,6 +61,8 @@
         "go1.x",
         "python3.8",
         "python3.9",
+        "python3.10",
+        "python3.11",
```

Comment:
What about 12?

Reply:
Weird dependency issue.  It requires a released version of the python plugin that has support for python 3.12 which isn't released yet.